src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final Random RANDOM = new Random();
+    private static final int BOUND_NUMBER = 100;
```

Comment:
```suggestion   
    private static final int BOUND_NUMBER = 100;
     private final Random random = new Random();
```

Reply:
made  getRandomColor() non -static.Made changes according to comment